tasks/withdraw.js

```diff
@@ -63,32 +95,63 @@ module.exports = async (callback) => {
         },
         []
     );
+    const withdrawTx = {
+        to: collectorAddress,
+        value: 0,
+        data: encodedWithdrawFunction
+    };
 
-    const transactions = [
-        {
-            to: collectorAddress,
-            value: 0,
-            data: encodedWithdrawFunction
+    let withdrawTxHash;
+    if (isMultisigWithdraw) {
+        const ethAdapterOwner1 = new Web3Adapter({
+            web3,
+            signerAddress: parsedSafeOwners[0]
+        });
+        const safeSdk = await Safe.create({
+            ethAdapter: ethAdapterOwner1,
+            safeAddress,
+            contractNetworks
+        });
+        const transactions = [withdrawTx];
+        const safeTransaction = await safeSdk.createTransaction(
+            ...transactions
+        );
+
+        // All the safe owners but the first one will sign the transaction before,
+        // while the parsedSafeOwners[0] will sign the transaction on execution.
+        for (let index = 1; index < parsedSafeOwners.length; index++) {
+            const owner = parsedSafeOwners[index];
+            await signWithAddress(web3, safeSdk, safeTransaction, owner);
         }
-    ];
-    const safeTransaction = await safeSdk.createTransaction(...transactions);
 
-    // All the owners but the first one will sign the transaction before,
-    // while the owners[0] will sign the transaction on execution.
-    for (let index = 1; index < owners.length; index++) {
-        const owner = owners[index];
-        await signWithAddress(web3, safeSdk, safeTransaction, owner);
+        /*
+         * TODO: We need to manually set the gasLimit in order to execute multisig tx on rsk regtest;
+         *       likely this value is going to change on testnet/mainnet
+         */
+        const safeTxGasLimit = '300000';
+        // owner1 execute (and implicitly signs) the transaction
+        const safeTxResponse = await safeSdk.executeTransaction(
+            safeTransaction,
+            {
+                gasLimit: safeTxGasLimit
+            }
+        );
+        withdrawTxHash = safeTxResponse.hash;
+    } else {
+        // TODO: We saw that '137410' is required in regtest, likely this value is going to change on testnet/mainnet
+        const gasRequired = '140000';
+        const tx = await web3.eth.sendTransaction({
+            ...withdrawTx,
+            from: '0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826',
```

Comment:
we need to change this one. we might need a new parameter, since the transaction must come from the collector owner

Reply:
My fault, I'm sorry, you're right, and I'd propose that we use directly the `owner` field in the collector instead of using an additional parameter ;-)

Reply:
superb! thanks